src/device/pnet_api.c

```diff
@@ -54,7 +54,8 @@ int pnet_init_only (pnet_t * net, const pnet_cfg_t * p_cfg)
    pf_dcp_exit (net); /* Prepare for re-init. */
    pf_dcp_init (net); /* Start DCP */
    pf_port_init (net);
-   net->interface.name_of_device_mode = PF_LLDP_NAME_OF_DEVICE_MODE_STANDARD;
+   net->interface.name_of_device_mode.mode =
```

Comment:
Maybe for clarity also:
`net->interface.name_of_device_mode.active = false;`

Reply:
👍